packages/http/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@stoplight/prism-core": "^3.1.1",
+    "@types/qs": "^6.5.3",
```

Comment:
We usually put the devDependencies (such as types) in the main package.json file of Prism.

Also, I miss `qs` package installed here, which we're going to need.

Reply:
Fixed 👍